torch/_functorch/_aot_autograd/dispatch_and_compile_graph.py

```diff
@@ -210,11 +210,21 @@ def _map_assigned_buffer_to_proxy(_mod, name, buffer):
                 colored=True,
             ),
         )
+
+        def fw_metadata_str():
+            return "\n\n".join(
+                [
+                    f"fw_metadata: {str(fw_metadata)}",
+                    f"maybe_subclass_meta: {str(maybe_subclass_meta)}",
+                    fw_module.print_readable(
+                        print_output=False, include_stride=True, include_device=True
+                    ),
+                ]
+            )
```

Comment:
Why put these together? Just output them as separate log entries. If you don't want to have to modify tlparse cli, dump the fw metadata as "artifact"

Reply:
you're completely right - I didn't realize that dumping using "artifact" allows us to add new artifacts without needing to change the tlparse cli. updated